src/pudl_archiver/orchestrator.py

```diff
@@ -194,10 +195,17 @@ async def run(self):
 
         await self._apply_changes()
         self.new_deposition = await self.depositor.get_record(self.new_deposition.id_)
-        await self._update_datapackage(resources)
+        new_datapackage, old_datapackage = await self._update_datapackage(resources)
         await self._apply_changes()
 
         if self.changed:
+            validation_result = self.downloader.validate_archive(
```

Comment:
I think `validation_success` would be a bit more readily interpretable here.

Reply:
Agreed.